hbase-server/src/test/java/org/apache/hadoop/hbase/util/TestRegionMover.java

```diff
@@ -83,12 +85,11 @@ public static void tearDownAfterClass() throws Exception {
   @Before
   public void setUp() throws Exception {
     // Create a pre-split table just to populate some regions
-    TableName tableName = TableName.valueOf("testRegionMover");
     Admin admin = TEST_UTIL.getAdmin();
-    if (admin.tableExists(tableName)) {
-      TEST_UTIL.deleteTable(tableName);
+    if (admin.tableExists(TABLE_NAME)) {
```

Comment:
this is race-y and won't work properly if tests are run in parallel. we should use a junit rule to get the name of the test method and use that for the table name.